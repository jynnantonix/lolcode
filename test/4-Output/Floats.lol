This test makes sure the print expression prints floating point decimals correctly.

HAI
	VISIBLE 1.00 2.345 6.789 MKAY?
KTHXBAI
