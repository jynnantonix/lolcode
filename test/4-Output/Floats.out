1.0 2.345 6.789 
